main.c

```diff
@@ -31,7 +31,6 @@
 #include <shlwapi.h>
 #include <wtsapi32.h>
 #include <prsht.h>
-#include <pbt.h>
```

Comment:
Actually PBT macros are now in winuser.h which is included through windows.h. So even if we use them later, we dont need this obsolete header.